docs/amber_script.md

```diff
@@ -134,8 +142,13 @@ Shaders can be added into pipelines with the `ATTACH` call. Shaders may be
 attached to multiple pipelines at the same time.
 
 ```
+  # The provided shader for ATTACH must _not_ be a 'multi' shader.
```

Comment:
What happens with   compute shader from OpenCL-C?  Do we declare it as a 'compute' type but then we would also need to give the entry-point name.

Reply:
Ah, so you're saying we need to combine ENTRY_POINT and ATTACH, that makes sense.